BTW, ci failed with python lint error 
```log
./src/pydolphinscheduler/tasks/sql.py:81:1: D400 First line should end with a period
```

TL;TR
run command `tox -e lint` or `flake8 .` could run the lint locally

and we have the way about run your code locally which you could see in https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-python/pydolphinscheduler/DEVELOP.md#test-your-code

--- a/dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/sql.py
+++ b/dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/sql.py
@@ -17,12 +17,15 @@
 
 """Task sql."""
 
+import logging
 import re
 from typing import Dict, Optional
 
 from pydolphinscheduler.constants import TaskType
 from pydolphinscheduler.core.database import Database
 from pydolphinscheduler.core.task import Task
+
+log = logging.getLogger(__file__)
 
 
 class SqlType:
@@ -78,13 +81,21 @@ class Sql(Task):
 
     @property
     def sql_type(self) -> int:
-        """If sql type is specified by parameter"""
+        """Judgement sql type, it will return the SQL type for type `SELECT` or `NOT_SELECT`.
+
+        If parameter :param:`param_sql_type` dot not specific, will use regexp to check
+        which type of the SQL is. But if parameter :param:`param_sql_type` is specific
+        will use the parameter overwrites the regexp way
+        """
         if (
             self.param_sql_type == SqlType.SELECT
             or self.param_sql_type == SqlType.NOT_SELECT
         ):
+            log.info(
+                "The sql type is specified by a parameter, with value %s",
+                self.param_sql_type,
+            )
             return self.param_sql_type
-        """Judgement sql type, use regexp to check which type of the sql is."""
         pattern_select_str = (
             "^(?!(.* |)insert |(.* |)delete |(.* |)drop "
             "|(.* |)update |(.* |)alter |(.* |)create ).*"

--- a/dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/sql.py
+++ b/dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/sql.py
@@ -83,8 +83,8 @@ class Sql(Task):
     def sql_type(self) -> int:
         """Judgement sql type, it will return the SQL type for type `SELECT` or `NOT_SELECT`.
 
-        If parameter :param:`param_sql_type` dot not specific, will use regexp to check
-        which type of the SQL is. But if parameter :param:`param_sql_type` is specific
+        If `param_sql_type` dot not specific, will use regexp to check
+        which type of the SQL is. But if `param_sql_type` is specific
         will use the parameter overwrites the regexp way
         """
         if (

--- a/dolphinscheduler-python/pydolphinscheduler/tests/tasks/test_sql.py
+++ b/dolphinscheduler-python/pydolphinscheduler/tests/tasks/test_sql.py
@@ -56,6 +56,8 @@ from pydolphinscheduler.tasks.sql import Sql, SqlType
         ("create table table_name2 (col1 int)", None, SqlType.NOT_SELECT),
         ("create table table_name2 (col1 int)", SqlType.SELECT, SqlType.SELECT),
         ("select 1", SqlType.NOT_SELECT, SqlType.NOT_SELECT),
+        ("create table table_name2 (col1 int)", SqlType.NOT_SELECT, SqlType.NOT_SELECT),
+        ("select 1", SqlType.SELECT, SqlType.SELECT),
     ],
 )
 @patch(

